plugins/woocommerce/src/Database/Migrations/CustomOrderTable/CLIRunner.php

```diff
@@ -198,8 +203,8 @@ public function migrate( $args = array(), $assoc_args = array() ) {
 
 			WP_CLI::debug(
 				sprintf(
-					// Translators: %1$d is the batch number, %2$f is time taken to process batch.
-					__( 'Batch %1$d (%2$d orders) completed in %3$f seconds', 'woocommerce' ),
+					// Translators: %1$d is the batch number, %2$d is the number of processed orders and %3$d is the execution time in seconds.
+					__( 'Batch %1$d (%2$d orders) completed in %3$d seconds', 'woocommerce' ),
```

Comment:
- Adjusted the translator comment.
- Changed the format of the execution time in seconds from float to integer.